maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -71980,6 +72042,19 @@
 	icon_state = "tcomms"
 	},
 /area/almayer/shipboard/weapon_room)
+"sSe" = (
+/obj/structure/sign/poster/blacklight{
+	pixel_y = 30
+	},
+/obj/structure/machinery/light/small{
+	dir = 8
+	},
+/obj/structure/reagent_dispensers/beerkeg/alt_dark,
```

Comment:
anchored = 1
density = 0
pixel off set it into the corner, reduce the contents to ~10-20%, better yet, write a quick and dirty subtype that changes the contents randomly via proc on init,